app/handlers/createHighlightVideo.py

```diff
@@ -35,7 +35,8 @@ def post(self):
         if status_code == 200:
             self.finish("Create Highlight Video")
         else:
-            raise tornado.web.HTTPError(reason=reason, status_code=status_code)
+            self.error_message = reason
```

Comment:
`self.error_message` is a property of BaseHandler (which this inherits from) and when the BaseHandler catches an error (like, say, the one thrown on the next line ;) ) it will use the `error_message` property to write that error message in the response JSON.